windows/CodePushNativeModule.cs

```diff
@@ -0,0 +1,291 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Windows.Web.Http;
+
+namespace CodePush.ReactNative
+{
+    internal class CodePushNativeModule : ReactContextNativeModuleBase
+    {
+        private CodePushLifecycleEventListener _codePushLifecycleEventListener = null;
```

Comment:
Sort these by variable name. Additionally, wouldn't the compiler set this to null by default?